gammapy/scripts/image_pipe.py

```diff
@@ -64,9 +67,12 @@ def make_counts(self, obs_id):
         obs = self.data_store.obs(obs_id=obs_id)
         events = obs.events.select_energy(self.energy_band)
         events = events.select_offset(self.offset_band)
-        counts_map.fill(events=events)
-        counts_map.data = counts_map.data.value
-        self.maps["counts"] = counts_map
+        if (len(events) != 0):
```

Comment:
For low energy range, there are runs without any events at these energies. I save the id number of these observations to not compute the bkg or the exposure for these runs...